aws/asg_test.go

```diff
@@ -85,6 +86,9 @@ func TestNukeAutoScalingGroups(t *testing.T) {
 	groupName := "aws-nuke-test-" + util.UniqueID()
 	createTestAutoScalingGroup(t, session, groupName)
 
+	// clean up ec2 instance created by the above call
+	defer nukeAllEc2Instances(session, findEC2InstancesByNameTag(t, session, groupName))
```

Comment:
Should you also call `nukeAllAutoScalingGroups`?

Reply:
Oh no, I'm deleting the ec2 instance created before the autoscaling group could be created. I'll update the variable name so it's less confusing